Makefile

```
@@ -747,6 +747,73 @@ else
 	rm -r $(TMP_DIR)
 endif
 
+# Download the osqueryd Linux executable from a pull request in osquery/osquery
+# and extract it into out-path.
+#
+# Usage:
+# make osqueryd-linux pr=8844 arch=amd64 out-path=.
+# make osqueryd-linux pr=8844 arch=arm64 out-path=.
+osqueryd-linux:
```

checkmake (0.3.2)

[warning] Target "osqueryd-linux" should be declared PHONY.

(phonydeclared)